libs/application/templates/children-residence-change/src/fields/MockData/applicant.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { Box, Text } from '@island.is/island-ui/core'
+import { InputController } from '@island.is/shared/form-fields'
+import { CRCFieldBaseProps } from '../../types'
+
+const applicantDefaults = {
+  name: 'Applicant Mockname',
+  address: 'Borgartún 26',
+  postalCode: '105',
+  city: 'Reykjavík',
+}
+
+const MockApplicant = ({ application, field }: CRCFieldBaseProps) => {
+  const { id } = field
+  const { answers } = application
+  const applicant = answers.mockData?.applicant
```

Comment:
We can use the default here instead of checking for it every time:
`const applicant = answers.mockData?.applicant || applicantDefaults`